packages/ui/Marquee/MarqueeBase.js

```diff
@@ -165,20 +166,22 @@ const MarqueeBase = kind({
 	},
 
 	computed: {
+		className: ({willAnimate, styler}) => styler.append({willAnimate}),
 		clientClassName: ({animating, willAnimate, styler}) => styler.join({
 			animate: animating,
 			text: true,
 			willAnimate
 		}),
-		clientStyle: ({alignment, animating, distance, overflow, rtl, speed}) => {
+		clientStyle: ({alignment, animating, distance, overflow, padding, rtl, speed}) => {
 			// If the components content directionality doesn't match the context, we need to set it
 			// inline
 			const direction = rtl ? 'rtl' : 'ltr';
 			const sideProperty = rtl ? 'left' : 'right';
 			const style = {
 				direction,
 				textAlign: alignment,
-				textOverflow: overflow
+				textOverflow: overflow,
+				'--ui-marquee-padding': padding
```

Comment:
Would this prevent developers from having different padding values for different marquees?